packages/@blockprotocol/graph/README.md

```diff
@@ -59,8 +59,6 @@ const graphService = new GraphEmbedderHandler({
 For React embedding applications, we provide a `useGraphEmbedderService` hook, which accepts a `ref` to an element, and optionally any additional constructor arguments you wish to pass.
 
 ```javascript
-import React from "react";
-
```

Comment:
Other parts of this example use `React.` btw – so they probably need updating

Reply:
Thanks, fixed!